scripts/zones/Southern_San_dOria/npcs/Machielle.lua

```diff
@@ -3,18 +3,13 @@
 --  NPC: Machielle
 -- Norvallen Regional Merchant
 -----------------------------------
-require("scripts/globals/events/harvest_festivals")
 local ID = require("scripts/zones/Southern_San_dOria/IDs")
-require("scripts/globals/conquest")
-require("scripts/globals/npc_util")
-require("scripts/globals/quests")
+require("scripts/globals/events/harvest_festivals")
+require("scripts/globals/shop")
```

Comment:
@TeoTwawki [got a little _too_ crazy with the require axe-murdering](https://github.com/project-topaz/topaz/commit/1dabc64278c58bd98955185114871a985d2323e6#diff-deeb89c2eb94a5679b39cc32db080314) 😉

Reply:
:scream: